SimPEG/Examples/DC_PseudoSection_Simulation.py

```diff
@@ -0,0 +1,220 @@
+from SimPEG import *
```

Comment:
This is the same as the one on the Examples Branch - @fourndo ? If so, I will remove it from this PR and we can take care of it on the Examples one